app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -1,65 +1,68 @@
-              color="primary"
-              icon
-              v-bind="attrs"
-              v-on="on"
-            >
-              <v-icon>publish</v-icon>
-              <input
-                class="d-none"
-                @change="loadFile"
-                ref="uploader"
-                type="file"
-                accept=".json"
-              />
-            </v-btn>
-          </template>
-          <span>{{ $t('trans.formDesigner.importDesign') }}</span>
-        </v-tooltip>
-      </v-col>
-      <!-- form name -->
-      <v-col cols="12" order="3">
-        <h3 v-if="name">{{ name }}</h3>
-      </v-col>
-      <!-- version number-->
-      <v-col cols="12" order="4">
-        <em
-          >{{ $t('trans.formDesigner.version') }} :
-          {{ this.displayVersion }}</em
-        >
-      </v-col>
-    </v-row>
-    <BaseInfoCard class="my-6">
+    <div :class="{ 'dir-rtl': isRTL }">
+      <v-row class="mt-6" no-gutters>
+        <!-- page title -->
+        <v-col cols="11">
+          <v-row>
+            <v-col cols="12">
+              <h1>
+                {{ $t('trans.formDesigner.formDesign') }}
+              </h1>
+              <h3 v-if="name">{{ name }}</h3>
+              <em
+                >{{ $t('trans.formDesigner.version') }} :
+                {{ this.displayVersion }}</em
+              >
+            </v-col>
+          </v-row>
+        </v-col>
+        <!-- buttons -->
+        <v-col cols="1">
+          <v-tooltip bottom>
+            <template #activator="{ on, attrs }">
+              <v-btn
+                class="mx-1"
+                @click="onExportClick"
+                color="primary"
+                icon
+                v-bind="attrs"
+                v-on="on"
+              >
+                <v-icon>get_app</v-icon>
+              </v-btn>
+            </template>
+            <span>{{ $t('trans.formDesigner.exportDesign') }}</span>
+          </v-tooltip>
+          <v-tooltip bottom>
+            <template #activator="{ on, attrs }">
+              <v-btn
+                class="mx-1"
+                @click="$refs.uploader.click()"
+                color="primary"
+                icon
+                v-bind="attrs"
+                v-on="on"
+              >
+                <v-icon>publish</v-icon>
+                <input
+                  class="d-none"
+                  @change="loadFile"
+                  ref="uploader"
+                  type="file"
+                  accept=".json"
+                />
+              </v-btn>
+            </template>
+            <span>{{ $t('trans.formDesigner.importDesign') }}</span>
+          </v-tooltip>
+        </v-col>
+        <!-- form name -->
+      </v-row>
+    </div>
+    <BaseInfoCard class="my-6" :class="{ 'dir-rtl': isRTL }">
       <h4 class="primary--text">
-        <v-icon class="mr-1" color="primary">info</v-icon
+        <v-icon :class="isRTL ? 'ml-1' : 'mr - 1'" color="primary">info</v-icon
```

Comment:
I think this should be `mr-1` without the spaces. It's really hard to see the difference, though, since it's only a few pixels!